### PR TITLE
feat: add-testing-lib

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/select/__tests__/index.test.js [hunk 1]
```diff
@@ -31,95 +30,105 @@ describe('Select', () => {
...
-    let dropdownWrapper = wrapper.find('Trigger');
-    expect(dropdownWrapper.prop('popupVisible')).toBe(true);
-    toggleOpen(wrapper);
+    expect(container.querySelectorAll('.ant-select-dropdown').length).toBe(1);
+    toggleOpen(container);
     expect(onDropdownVisibleChange).toHaveBeenLastCalledWith(false);
-    expect(dropdownWrapper.prop('popupVisible')).toBe(true);
+    expect(container.querySelectorAll('.ant-select-dropdown').length).toBe(1);
 
-    wrapper.setProps({ open: false });
```
Comment (heiyu4585) [inline components/select/__tests__/index.test.js:77]: 这个地方我有点拿不准 wrapper.setProps后,如何验证 prop ,我参考了[testing-lib](https://testing-library.com/docs/example-update-props) 文档和 antd 其他组件testing部分(没找到类似的),

Comment (heiyu4585) [reply components/select/__tests__/index.test.js:77]: ` expect(container.querySelectorAll('.ant-select-dropdown').length).toBe(1);` 未体现出变化

**Comment by zombieJ** [comment]
继续开 PR？

**Comment by heiyu4585** [comment]
> 继续开 PR？

https://github.com/ant-design/ant-design/pull/35633

**Pull request merged**